client/config.go

```diff
@@ -3,6 +3,8 @@ package client
 import (
 	"github.com/spf13/pflag"
 	"github.com/spf13/viper"
+	"os/user"
```

Comment:
https://github.com/mitchellh/go-homedir?

Oh now that I've looked ahead I see that you're expanding `~user/blah` as well as `~/blah` and I think go-homedir only expands the `~/` case